<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>publications | Jiseon Min</title> <meta name="author" content="Jiseon Min"> <meta name="description" content=""> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.css"> <link rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?19f3075a2d19613090fe9e16b564e1fe" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://jiseonmin.github.io/publications/"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?e74e74bf055e5729d44a7d031a5ca6a5" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js?96d6b3e1c3604aca8b6134c7afdd5db6"></script> <script src="/assets/js/dark_mode.js?9b17307bb950ffa2e34be0227f53558f"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Jiseon </span>Min</a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about</a> </li> <li class="nav-item "> <a class="nav-link" href="/blog/">blog</a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description"></p> </header> <article> <div class="publications"> <h2 class="bibliography">2022</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="min_spatial_2022" class="col-sm-8"> <div class="title">Spatial structure alters the site frequency spectrum produced by hitchhiking</div> <div class="author"> Jiseon Min, Misha Gupta, Michael M Desai, and Daniel B Weissman</div> <div class="periodical"> <em>Genetics</em>, Nov 2022 </div> <div class="periodical"> </div> <div class="links"> <a href="https://academic.oup.com/genetics/article/doi/10.1093/genetics/iyac139/6696215" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> </div> </div> </li></ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="johnson_phenotypic_2021" class="col-sm-8"> <div class="title">Phenotypic and molecular evolution across 10,000 generations in laboratory budding yeast populations</div> <div class="author"> Milo S Johnson, Shreyas Gopalakrishnan, Juhee Goyal, Megan E Dillingham, Christopher W Bakerlee, Parris T Humphrey, Tanush Jagdish, Elizabeth R Jerison, Katya Kosheleva, Katherine R Lawrence, Jiseon Min, Alief Moulana, Angela M Phillips, Julia C Piper, Ramya Purkanti, Artur Rego-Costa, Michael J McDonald, Alex N Nguyen Ba, and Michael M Desai</div> <div class="periodical"> <em>eLife</em>, Jan 2021 </div> <div class="periodical"> Publisher: eLife Sciences Publications, Ltd </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Laboratory experimental evolution provides a window into the details of the evolutionary process. To investigate the consequences of long-term adaptation, we evolved 205 Saccharomyces cerevisiae populations (124 haploid and 81 diploid) for ~10,000 generations in three environments. We measured the dynamics of fitness changes over time, finding repeatable patterns of declining adaptability. Sequencing revealed that this phenotypic adaptation is coupled with a steady accumulation of mutations, widespread genetic parallelism, and historical contingency. In contrast to long-term evolution in E. coli, we do not observe long-term coexistence or populations with highly elevated mutation rates. We find that evolution in diploid populations involves both fixation of heterozygous mutations and frequent loss-of-heterozygosity events. Together, these results help distinguish aspects of evolutionary dynamics that are likely to be general features of adaptation across many systems from those that are specific to individual organisms and environmental conditions.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="levien_non-genetic_2021" class="col-sm-8"> <div class="title">Non-genetic variability in microbial populations: survival strategy or nuisance?</div> <div class="author"> Ethan Levien, Jiseon Min, Jane Kondev, and Ariel Amir</div> <div class="periodical"> <em>Reports on Progress in Physics</em>, Nov 2021 </div> <div class="periodical"> Publisher: IOP Publishing </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The observation that phenotypic variability is ubiquitous in isogenic populations has led to a multitude of experimental and theoretical studies seeking to probe the causes and consequences of this variability. Whether it be in the context of antibiotic treatments or exponential growth in constant environments, non-genetic variability has significant effects on population dynamics. Here, we review research that elucidates the relationship between cell-to-cell variability and population dynamics. After summarizing the relevant experimental observations, we discuss models of bet-hedging and phenotypic switching. In the context of these models, we discuss how switching between phenotypes at the single-cell level can help populations survive in uncertain environments. Next, we review more fine-grained models of phenotypic variability where the relationship between single-cell growth rates, generation times and cell sizes is explicitly considered. Variability in these traits can have significant effects on the population dynamics, even in a constant environment. We show how these effects can be highly sensitive to the underlying model assumptions. We close by discussing a number of open questions, such as how environmental and intrinsic variability interact and what the role of non-genetic variability in evolutionary dynamics is.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="barber_modeling_2021" class="col-sm-8"> <div class="title">Modeling the impact of single-cell stochasticity and size control on the population growth rate in asymmetrically dividing cells</div> <div class="author"> Felix Barber, Jiseon Min, Andrew W. Murray, and Ariel Amir</div> <div class="periodical"> <em>PLOS Computational Biology</em>, Jun 2021 </div> <div class="periodical"> Publisher: Public Library of Science </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Microbial populations show striking diversity in cell growth morphology and lifecycle; however, our understanding of how these factors influence the growth rate of cell populations remains limited. We use theory and simulations to predict the impact of asymmetric cell division, cell size regulation and single-cell stochasticity on the population growth rate. Our model predicts that coarse-grained noise in the single-cell growth rate λ decreases the population growth rate, as previously seen for symmetrically dividing cells. However, for a given noise in λ we find that dividing asymmetrically can enhance the population growth rate for cells with strong size control (between a “sizer” and an “adder”). To reconcile this finding with the abundance of symmetrically dividing organisms in nature, we propose that additional constraints on cell growth and division must be present which are not included in our model, and we explore the effects of selected extensions thereof. Further, we find that within our model, epigenetically inherited generation times may arise due to size control in asymmetrically dividing cells, providing a possible explanation for recent experimental observations in budding yeast. Taken together, our findings provide insight into the complex effects generated by non-canonical growth morphologies.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="min_transport_2021" class="col-sm-8"> <div class="title">A transport approach to relate asymmetric protein segregation and population growth</div> <div class="author"> Jiseon Min, and Ariel Amir</div> <div class="periodical"> <em>Journal of Statistical Mechanics: Theory and Experiment</em>, Jul 2021 </div> <div class="periodical"> Publisher: IOP Publishing and SISSA </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Many unicellular organisms allocate their key proteins asymmetrically between the mother and daughter cells, especially in a stressed environment. A recent theoretical model is able to predict when the asymmetry in segregation of key proteins enhances the population fitness, extrapolating the solution at two limits where the segregation is perfectly asymmetric (asymmetry a = 1) and when the asymmetry is small (0 ⩽ a ≪ 1) (Lin et al 2019 Phys. Rev. Lett. 122 068101). We generalize the model by introducing stochasticity and use a transport equation to obtain a self-consistent equation for the population growth rate and the distribution of the amount of key proteins. We provide two ways of solving the self-consistent equation: numerically by updating the solution for the self-consistent equation iteratively and analytically by expanding moments of the distribution. With these more powerful tools, we can extend the previous model by Lin et al (2019 Phys. Rev. Lett. 122 068101) to include stochasticity to the segregation asymmetry. We show the stochastic model is equivalent to the deterministic one with a modified effective asymmetry parameter (a eff). We discuss the biological implication of our models and compare with other theoretical models.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2020</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="10.7554/eLife.61771" class="col-sm-8"> <div class="title">Dynamics of diffusive cell signaling relays</div> <div class="author"> Paul B Dieterle, Jiseon Min, Daniel Irimia, and Ariel Amir</div> <div class="periodical"> <em>eLife</em>, Dec 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>In biological contexts as diverse as development, apoptosis, and synthetic microbial consortia, collections of cells or subcellular components have been shown to overcome the slow signaling speed of simple diffusion by utilizing diffusive relays, in which the presence of one type of diffusible signaling molecule triggers participation in the emission of the same type of molecule. This collective effect gives rise to fast-traveling diffusive waves. Here, in the context of cell signaling, we show that system dimensionality – the shape of the extracellular medium and the distribution of cells within it – can dramatically affect the wave dynamics, but that these dynamics are insensitive to details of cellular activation. As an example, we show that neutrophil swarming experiments exhibit dynamical signatures consistent with the proposed signaling motif. We further show that cell signaling relays generate much steeper concentration profiles than does simple diffusion, which may facilitate neutrophil chemotaxis.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2019</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="lin_optimal_2019" class="col-sm-8"> <div class="title">Optimal Segregation of Proteins: Phase Transitions and Symmetry Breaking</div> <div class="author"> Jie Lin, Jiseon Min, and Ariel Amir</div> <div class="periodical"> <em>Physical Review Letters</em>, Feb 2019 </div> <div class="periodical"> Publisher: American Physical Society </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Asymmetric segregation of key proteins at cell division—be it a beneficial or deleterious protein—is ubiquitous in unicellular organisms and often considered as an evolved trait to increase fitness in a stressed environment. Here, we provide a general framework to describe the evolutionary origin of this asymmetric segregation. We compute the population fitness as a function of the protein segregation asymmetry a, and show that the value of a which optimizes the population growth manifests a phase transition between symmetric and asymmetric partitioning phases. Surprisingly, the nature of phase transition is different for the case of beneficial proteins as opposed to deleterious proteins: a smooth (second order) transition from purely symmetric to asymmetric segregation is found in the former, while a sharp transition occurs in the latter. Our study elucidates the optimization problem faced by evolution in the context of protein segregation, and motivates further investigation of asymmetric protein segregation in biological systems.</p> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom"> <div class="container mt-0"> © Copyright 2023 Jiseon Min. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?7b30caa5023af4af8408a472dc4e1ebb"></script> <script defer src="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.js"></script> <script src="/assets/js/no_defer.js?d633890033921b33e0ceb13d22340a9c"></script> <script defer src="/assets/js/common.js?acdb9690d7641b2f8d40529018c71a01"></script> <script defer src="/assets/js/copy_code.js?c9d9dd48933de3831b3ee5ec9c209cac" type="text/javascript"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>